tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -15,6 +15,24 @@
       "IncludeOnlyTargetFrameworks": ["net462"]
     }]
   },
+  {
+    "IntegrationName": "AwsSns",
+    "SampleProjectName": "Samples.AWS.SQS",
+    "NugetPackageSearchName": "AWSSDK.SimpleNotificationService",
+    "MinVersion": "3.0.0",
+    "MaxVersionExclusive": "3.8.0",
+    "SpecificVersions": [
+      "3.1.*",
+      "3.3.*",
+      "3.*.*"
+    ],
+    "VersionConditions": [
+      {
+        "MaxVersionExclusive": "3.3.0",
+        "IncludeOnlyTargetFrameworks": ["net462"]
```

Comment:
There are quite a large number of package version changes in this PR. I think you will need to rebase this PR on master to accept the net462=>net471 changes made in https://github.com/DataDog/dd-trace-dotnet/pull/4057